mono/mini/mini-exceptions-native-unwinder.c

```diff
@@ -28,7 +30,7 @@
 
 #define UNW_LOCAL_ONLY
 #undef _U /* ctype.h apparently defines this and it screws up the libunwind headers. */
-#include "android-libunwind/libunwind.h"
+#include "../../external/android-libunwind/include/libunwind.h"
```

Comment:
This hardcodes path locations. I think it would be better to specify the directory as an include path and leave the include as it was.